crates/astria-sequencer-relayer/src/relayer/write/conversion.rs

```diff
@@ -575,4 +576,12 @@ mod tests {
         let payload = input.try_into_payload().unwrap();
         assert_eq!(2, payload.num_blobs());
     }
+
+    #[tokio::test]
+    async fn should_compress() {
+        let mut next_submission = NextSubmission::new(include_all_rollups());
+        next_submission.try_add(block(1)).unwrap();
+        let submission = next_submission.take().await.unwrap();
+        assert!(submission.compression_ratio() > 1.0);
```

Comment:
I mean, this is not guaranteed always. In principle the compression *could* make things worse.

I don't think a unit test here is strictly necessary.

Reply:
Yeah - fair enough.  It did seem a little overkill to me too, although if we updated something in the future which caused this test to start failing, we could always change the test to use data which is guaranteed to compress.

Either way - I'll remove the test, as it's unlikely we'll accidentally cause a regression in the `compression_ratio` method.

Reply:
Removed in [210daca](https://github.com/astriaorg/astria/pull/1075/commits/210daca1a6ea79387378df6959ca0e6d41a133c7).